src/Utilities/Workspaces/Workspaces.Utilities.projitems

```diff
@@ -13,8 +13,8 @@
     <Compile Include="$(MSBuildThisFileDirectory)SolutionChangeAction.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)SyntaxGeneratorExtensions.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)SyntaxNodeExtensions.cs" />
-  </ItemGroup>
+  </ItemGroup>  
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis" Version="$(MicrosoftCodeAnalysisVersion)" />
+    <PackageReference Include="Microsoft.CodeAnalysis.Workspaces.Common" Version="$(MicrosoftCodeAnalysisVersion)" />
```

Comment:
I couldn't find how to have a conditional include here that would say include the C# workspace nuget if the project name (or assembly name) contains `CSharp`, include VB for vb and fallback to the common.

@sharwell any idea?

Reply:
IMO the **projitems** should never have `ProjectReference`, but other people feel differently about this.

Reply:
I think the most common expectation would be **Workspaces.Utilities** references language-agnostic only, and **Workspaces.Utilities.CSharp** references the C# assemblies (i.e. split the shared projects the same way normal assemblies are split).

Reply:
I also prefer not to have references in shared projects and to be honest with new project system I have stopped using shared project, I simply add manual reference to the files/folder I want to share. But I also understand how it makes it easier to share this common dependency.

Regarding `Workspaces.Utilities.CSharp` we currently don't have it and don't have C# specific code. I was thinking about a `Choose` just to help with avoiding the reference on each project.

If you all agree then I suggest to keep what I have done in this PR, otherwise I am open to suggestion :)

Reply:
I have no preference here - I am fine with whatever is easier for you @Evangelink :-)